app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -785,6 +787,17 @@ public void onClick(DialogInterface dialog, int which) {
                             case 1:
                                 ServiceHelper.addLink(ReadArticleActivity.this, url);
                                 break;
+                            case 2:
+                                ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
+                                ClipData clip = ClipData.newRawUri("wallabag", Uri.parse(url));
+                                clipboard.setPrimaryClip(clip);
+                                break;
+                            case 3:
+                                Intent sendIntent = new Intent();
+                                sendIntent.setAction(Intent.ACTION_SEND);
+                                sendIntent.putExtra(Intent.EXTRA_TEXT, url);
+                                sendIntent.setType("text/plan");
```

Comment:
i guess it should be text/plain.

Reply:
Also there is `shareArticle()` for this.

Reply:
So should I leave it in or not?

Reply:
Can you please reuse `shareArticle()` as @di72nn suggested?

Reply:
I use it now.